embed/script.js

```diff
@@ -0,0 +1,252 @@
+function h() {
+  var type = arguments[0] || 'div';
+  var attributes = arguments[1] || {};
+  var children = Array.prototype.slice.call(arguments, 2);
+  var el = document.createElement(type);
+
+  if (children[0] instanceof Array) {
+    children = children[0];
+  }
+
+  Object.keys(attributes)
+    .map(function(key) {
```

Comment:
Are we using the new fat arrow syntax: `(key) => { }` or old school style: `function() { }`

We should just stick with one of these styles (I would suggest the first one) as it greatly improves readability

Reply:
I'm seeing both styles across the codebase - so maybe a find & replace is in order!

Reply:
Support for fat arrow syntax is still poor so I need to transpile it. We want to keep the code in `embed.js` loadable by any/most browser.

Reply:
Moved all to `function`s.